index.js

```diff
@@ -127,25 +145,20 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventory = userData[userIdentification].inventory;
+    let cardData = Object.values(userInventory);
```

Comment:
small thing -- it's totally reasonable and often preferable to do what you're doing with creating a variable here. Since we only use `cardData` in one place, we could also just replace `cardData` with `Object.values(userInventory)`. Up to you -- even though this adds an additional line, both are fine if naming a variable provides more clarity than simplifying the code. As mentioned before, you always want to reduce the number of lines unless it adds clarity.